dnf/base.py

```diff
@@ -858,8 +858,12 @@ def do_transaction(self, display=()):
                 for descr in tserrors:
                     errstring += '  %s\n' % ucd(descr)
 
-                raise dnf.exceptions.Error(errstring + '\n' +
-                                        self._trans_error_summary(errstring))
+                summary = self._trans_error_summary(errstring)
+                if summary:
+                    errstring += '\n' + summary
+
+                raise dnf.exceptions.Error(errstring)
```

Comment:
Unrelated rant: Seems like the summary was meant to be a general summary, but it only deals with disk space errors so far. So saying "Error Summary" when what follows doesn't sum up all the errors, but just some of them, is somewhat misleading...

Reply:
yes, but if we changed the headline than we should also change the function name and maybe some variables? I am not sure if its worth it. Even though it's an underscore function.

Reply:
I don't think it's worth changing right now. The patch fixes the issue mentioned in the bug, so it's less misleading now. Tests are passing locally, so I'll merge this.